astropy/coordinates/sky_coordinate.py

```diff
@@ -688,6 +688,69 @@ def separation_3d(self, other):
         distval = (dx.value ** 2 + dy.value ** 2 + dz.value ** 2) ** 0.5
         return Distance(distval, dx.unit)
 
+    def spherical_offsets_to(self, tocoord):
+        r"""
+        Computes angular offsets to go *from* this coordinate *to* another.
+
+        Parameters
+        ----------
+        tocoord : `~astropy.coordinates.BaseCoordinateFrame`
+            The coordinate to offset to.
+
+        Returns
+        -------
+        lon_offset : `~astropy.coordinates.Angle`
+            The angular offset in the longitude direction (i.e., RA for
```

Comment:
Same here, I would remove the `(i.e., RA for equatorial coordinates)` part ... it's pretty clear what longitude and latitude is, no? (but if you disagree, leave as is, it's not important)